style.css

```diff
@@ -1,19 +1,20 @@
 :root {
     --black: #000;
     --white: #fff;
+    --card-title: rgba(148, 187, 233, 1);
+    --background: linear-gradient(90deg, rgba(238, 174, 202, 1) 0%, rgba(234, 175, 203, 1) 0%, rgba(148, 187, 233, 1) 100%);
+    --card-text: #012169;
+    --card-border: grey;
```

Comment:
These are not generic variables but specific to the card. Move them inside the `.repo-card` class.

Same with th `card-title` on line 4

Reply:
What I meant here is to move the CSS variables, not delete them and put the value directly.